src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -870,7 +870,9 @@ def main(context):
     elif type == 'json':
         return context.res.json({"type": "This is a JSON response"})
     elif type == 'binary':
-        return context.res.binary(InputFile.from_path('file.png'))
+        with open('file.png', 'rb') as file:
+            file_contents = file.read()
+        return context.res.binary(file_contents)
```

Comment:
is `file_contents` still in scope here?

Reply:
It is indeed and accessible to the `context.res` for `binary`.